File: 프로그래머스/0/181890. 왼쪽 오른쪽/왼쪽 오른쪽.java
import java.util.Arrays;
class Solution {
    public String[] solution(String[] str_list) {
        
        String[] result = {};
        
        for(int i=0; i < str_list.length; i++){
            String s = str_list[i];
            
            if(s.equals("l")){
                result = Arrays.copyOfRange(str_list,0,i);
                break;
            }
            else if(s.equals("r")){
                result = Arrays.copyOfRange(str_list,i+1,str_list.length);
                break;
            }
        }
        
        return result;
    }
}

File: 프로그래머스/0/181890. 왼쪽 오른쪽/왼쪽 오른쪽.py
def solution(str_list):
    # 먼저 나오는 문자열이 "l"이라면, 해당 문자열을 기준으로 왼쪽에 있는 문자열 출력
    # 먼저 나오는 문자열이 "r"이라면, 해당 문자열을 기준으로 오른쪽에 있는 문자열 출력
    # "l"이나 "r"이 없다면 빈 리스트
    
    result = []
    
    for i in range(len(str_list)):
        if str_list[i] == 'l':
            result = str_list[:i]
            break
        elif str_list[i] == 'r':
            result = str_list[i+1:]
            break
    
    return result